docs/developers-guide/decorators/json-schema-enabled decorator.md

```diff
@@ -0,0 +1,93 @@
+# The json-schema-enabled Decorator
+This document explains how the json-schema-enabled decorator works and how it is applied to:
+* values
```

Comment:
Values are not used for JSON schema generation.

Reply:
Fixed now